specifications/api/swagger.yaml

```diff
@@ -468,6 +468,22 @@ x-stakePoolMargin: &stakePoolMargin
   description: |
       Variable margin on the total reward given to an operator before splitting rewards between stakeholders.
 
+x-stakePoolSaturation: &stakePoolSaturation
+  type: number
+  minimum: 0
+  description: |
+    Saturation-level of the pool based on the desired number of pools aimed by the network.
+    A value above `1` indicates that the pool is saturated.
+  example: 0.74
+
+x-stakePoolDesirability: &stakePoolDesirability
+  type: number
+  minimum: 0
+  description: |
+    How desirable / attractive a pool is. To determine a pool's _rank_, we order pools by decreasing desirability.
```

Comment:
I have already said this, but I'd be concerned about people not realising that this does not take oversaturation into account.

**If** `poolMemberRewards` should ultimately be used for ordering, and ranking, the last sentence would be somewhat misleading too.

Reply:
No. The rank of a pool is and will always be defined as the desirability. The pool member rewards is simply an additional view that users may use to choose their pool.

Reply:
> rank of a pool is and will always be defined as the desirability

"Fine". But then "rank" is a very technical term, and has nothing to do with the order it should ultimately be displayed in Daedalus — assuming ordering by member rewards. Why does users of the API need to know this rather arcane definition of "rank"?

Fine if we change it later of course.

I just am being slightly picky here on purpose because I am not completely sure we're on the same page.

In a similar notion, FYI -> https://input-output-rnd.slack.com/archives/CCRB7BU8Y/p1579613854051400